webserver/shared/models/DelegationForPool.ts

```diff
@@ -0,0 +1,13 @@
+import { Address } from "./Address";
+import { Pool } from "./Pool";
+
+export type DelegationForPoolRequest = {
+  pools: Pool[];
+  range: { minSlot: number, maxSlot: number }
+};
+
+export type DelegationForPoolResponse = {
+    credential: Address;
```

Comment:
Not sure yet what's the best way to show this, currently it's the credential in hex